htmd/projections/metriccoordinate.py

```diff
@@ -49,7 +63,8 @@ def __init__(self, refmol, atomsel, trajalnstr='protein and name CA', refalnstr=
         self.pbc = pbc
 
     def _precalculate(self, mol):
```

Comment:
is this ever used?

Reply:
Yes! It's called by `Metric` if the simlist has a unique structure. This is the only case in which you should precalculate atomselections as in other cases like in `MetricCoords` the atomselection you make on `mol` might not apply on all systems with different number of atoms and hence `MetricCoords` would crash.

Reply:
how did it pass the tests then?

Reply:
We should be testing simlists

Reply:
Yes. Would be nice to have a test which tests against a system with mutations (hence different number of atoms) like bar/bar

Reply:
For future reference. Here is the place the `precalculate` is called: https://github.com/Acellera/htmd/blob/master/htmd/projections/metric.py#L135 and here it is defined as obligatory for all `Projection` classes that `Metric` uses https://github.com/Acellera/htmd/blob/master/htmd/projections/projection.py#L23